src/client/version/client.go

```diff
@@ -42,6 +42,15 @@ func PrettyPrintVersion(version *pb.Version) string {
 	return result
 }
 
+// This returns true if Pachyderm is at least at the given version. This allows
+// us to gate backwards-incompatible features on release boundaries.
+//
+// If version.IsAtLeast() is called anywhere in the code with an old version, it
+// can safely be assumed to be "true" and removed
```

Comment:
I'm not sure what this means, is there a way to reword this?

Reply:
Sure, though I'm not sure what the best way to express the idea might be. Basically the idea is that if we're on Pachyderm 1.9.3, and you're a Pachyderm engineer and you see
```
if version.IsAtLeast(1, 9) {
  /* backwards-incompatible code */
}
```
then you can just replace this with
```
/* backwards-incompatible code */
```
This function is basically supposed to be a simple way to gate stuff (rather than what we've been doing, which is to file an issue for backwards-incompatible changes and hoping we remember to fix them between releasing the first RC for a new version and releasing the version itself).

Part of me thinks I should just get rid of the `If version.IsAtLeast()` paragraph, since the fact it's highlighting is pretty obvious, so it might be creating more confusion than it's clearing, but I'm curious what you think

Reply:
Yeah, I think the paragraph just adds confusion